server/src/main/java/no/ntnu/klubbhuset/domain/User.java

```diff
@@ -32,10 +35,13 @@ public void addSecurityGroup(SecurityGroup securityGroup) {
     @Id
     private String email;
 
-    @JsonIgnore
```

Comment:
Wont this expose the password in the token?

Reply:
JWT leser vel kun info fra usersecurityroles tabellen?